src/bin.js

```diff
@@ -15,6 +15,7 @@ simple:                $0 ./src/index.js
 multiple entry:        $0 ./src/index.js ./src/entry2.js
 with custom directory: $0 ./src/index.js --dir /path/to/other/folder
 with  exclude:         $0 ./src/index.js --exclude tests  utils/webpack
+without report server: $0 ./src/index.js --no-report-server utils/webpack
```

Comment:
we also need to add the required changes for this option in the `src/cli/options.js`

Reply:
Done